setenv.sh

```diff
@@ -6,9 +6,24 @@ export KDBCODE=${TORQHOME}/code
 export KDBLOG=${TORQHOME}/logs
 export KDBHTML=${TORQHOME}/html
 export KDBLIB=${TORQHOME}/lib
+export KDBAPPCODE=${TORQHOME}/code/tick
```

Comment:
This isn't right I think - should have the /tick on the end

Reply:
It does have /tick on the end - did you mean shouldn't? If so, I changed this because for the feed process.csv loads the script is with using `${KDBAPPCODE}/feed.q`. This was how it was set up in the process.csv I was given.

I will change it so `export KDBAPPCODE=${TORQHOME}/code` and the feed loads `${KDBAPPCODE}/tick/feed.q`